tests/handler/test_service_handler_decorator_results_in_correctly_functioning_operation_factories.py

```diff
@@ -21,13 +21,19 @@
     service_handler,
     sync_operation,
 )
-from nexusrpc.handler._core import collect_operation_handler_factories_by_method_name
 from nexusrpc.handler._decorators import operation_handler
+from nexusrpc.handler._operation_handler import (
+    collect_operation_handler_factories_by_method_name,
+)
+
+
+@dataclass_transform()
+class _BaseTestCase:
+    pass
```

Comment:
I'm using `dataclass_transform` in tests to allow me to create classes of the form

```python
class MyTestCaseClass:
    some_attr: Foo
```

`MyTestCaseClass` is never instantiated in my tests; it's syntactic/ergonomic convenience for a module to hold some nexus service definition.

My expectation was that `@dataclass_transform` could be used directly on the class, but it appears to only take effect when applied to a parent class.